Problems with installing package - missing file documentation.yml

Hello,  I read article on [medium](https://medium.freecodecamp.org/how-to-write-powerful-schemas-in-javascript-490da6233d37) and I wanted to give a try however I have a problem with installing package.

After running `npm i --save schm` I encountered error:
```

> schm@0.3.6 postinstall /directory/repo/node_modules/schm
> npm run build


> schm@0.3.6 prebuild /directory/repo/node_modules/schm
> npm run docs && npm run clean


> schm@0.3.6 docs /directory/repo/node_modules/schm
> documentation readme src --section=API --pe flow -c documentation.yml

{ Error: ENOENT: no such file or directory, open '/directory/repo/node_modules/schm/documentation.yml'
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/directory/repo/node_modules/schm/documentation.yml' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! schm@0.3.6 docs: `documentation readme src --section=API --pe flow -c documentation.yml`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the schm@0.3.6 docs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /directory/.npm/_logs/2018-03-22T08_43_11_423Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! schm@0.3.6 prebuild: `npm run docs && npm run clean`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the schm@0.3.6 prebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /directory/.npm/_logs/2018-03-22T08_43_11_451Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! schm@0.3.6 postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the schm@0.3.6 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /directory/.npm/_logs/2018-03-22T08_43_11_666Z-debug.log
```

If you can help me, I would be grateful, thanks!

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "lint": "eslint packages/*/{src,test}",
     "docs": "lerna exec -- npm run docs",
     "flow": "flow",
-    "postinstall": "lerna bootstrap"
+    "postinstall": "lerna bootstrap && lerna exec -- npm run build"
   },
   "jest": {
     "testEnvironment": "node",

--- a/packages/schm-computed/package.json
+++ b/packages/schm-computed/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

--- a/packages/schm-express/package.json
+++ b/packages/schm-express/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

--- a/packages/schm-koa/package.json
+++ b/packages/schm-koa/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

--- a/packages/schm-methods/package.json
+++ b/packages/schm-methods/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

--- a/packages/schm-mongo/package.json
+++ b/packages/schm-mongo/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist --copy-files",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

--- a/packages/schm-translate/package.json
+++ b/packages/schm-translate/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

--- a/packages/schm/package.json
+++ b/packages/schm/package.json
@@ -26,7 +26,6 @@
     "clean": "rimraf dist",
     "prebuild": "npm run docs && npm run clean",
     "build": "babel src -d dist --copy-files",
-    "postinstall": "npm run build",
     "prepublishOnly": "npm run lint && npm test && npm run build"
   },
   "jest": {

